commands/store.js

```diff
@@ -94,7 +94,7 @@ class Store extends Command {
             }
             await this.client.db.giveUserCurrency(connection, message.guild.id, message.author.id, returnPrice * amount);
             await connection.query('COMMIT');
-            return message.channel.send(`${message.author} You have sold ${amount}x ${storeItem.name} :tada:`);
+            return message.channel.send(`${message.author} You have sold ${amount}x ${storeItem.name} :tada:\nYou now have <:blobcoin:386630453224013824> ${userData.currency}.`);
           } else
```

Comment:
Ditto, userData is not in scope.

Reply:
As for here, `deducted` contains the other side of the deal and isn't as useful to us. We can instead store the value from `giveUserCurrency`, which like `takeUserCurrency`, returns a `currency` field:
```js
const updatedUser = await this.client.db.giveUserCurrency(...);
...
`You now have ${updatedUser.currency}`
```
etc